KOA/clanwars/models.py

```diff
@@ -1,3 +1,10 @@
 from django.db import models
 
 # Create your models here.
+# class Status(models.Model):
```

Comment:
I don't really understand what does this commented example explain?
Havn't used django in a while so might be why, but if we can have a short explanation of this class if could be cool! :)

Reply:
I *was* using it but it turns out I didn't need it so I removed it to keep things "light" for now.
It basically creates a model in the database, but I think we are still debating whether we need a database right now?